HHS/OPRE-OPS
# budget lines workflow from PLANNED to EXECUTING

## What changed

Workflow integration for budgetlines from PLANNED TO EXECUTING

## Issue

#677 

## How to test

1. Submit Agreement 1 for approval.
2. Review BLIs Status. Should show "Under Review".
3. Goto Agreements Approval link and select approve icon for Agreement 1.
4. Approve the changes.
5. Review BLIs Status. Should show "Planned".
6. Submit Agreement 1 for approval.
7. Review BLIs Status. Should show "Under Review".
8. Goto Agreements Approval link and select approve icon for Agreement 1.
9. Approve the changes.
10. Review BLIs Status. Should show "Executing".

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
